main/main.go

```diff
@@ -428,14 +444,14 @@ func createApplicationRelayersForSourceChain(
 			logger,
 			metrics,
 			network,
-			messageCreator,
```

Comment:
Looks like `messageCreator` can be removed as a param for `createApplicationRelayersForSourceChain` and `createApplicationRelayers`